x-pack/plugins/security_solution/public/management/components/artifact_entry_card/artifact_entry_card.tsx

```diff
@@ -46,6 +48,11 @@ export interface ArtifactEntryCardProps extends CommonArtifactEntryCardProps {
   hideComments?: boolean;
 }
 
+export interface ArtifactEntryCardDecoratorProps {
+  item: MaybeImmutable<AnyArtifact>;
+  testIdPrefix?: string;
```

Comment:
the normal pattern has been to use `'data-test-subj'`. Maybe consider using that as the prop name instead of `testIdPrefix`?

Reply:
b3f70b050123b00b78e1e248ab90310afd53d4d5